### ISSUE TITLE
No such file or directory: '/export/apple_health_export/workout-routes/'

### ISSUE THREAD

**Issue by scarroll32** [k0rventen/apple-health-grafana#2]
This looks like an amazing app, but I am having trouble getting it started.

When I run the ingester I get:

> No such file or directory: '/export/apple_health_export/workout-routes/'

```
❯ docker-compose up ingester
[+] Running 9/9
 ⠿ ingester Pulled                                                         4.6s
   ⠿ 63b65145d645 Pull complete                                            0.8s
   ⠿ 781eddb6f342 Pull complete                                            1.0s
   ⠿ 1ae1f1ea756f Pull complete                                            1.3s
   ⠿ 5b3e7c82c61d Pull complete                                            1.4s
   ⠿ 59b6a1c4fe6c Pull complete                                            1.6s
   ⠿ 07c7907083b1 Pull complete                                            1.6s
   ⠿ 48a8ca346aeb Pull complete                                            2.6s
   ⠿ 99e3fc1df906 Pull complete                                            2.7s
[+] Running 1/1
 ⠿ Container apple-health-grafana-ingester-1  Created                      0.1s
Attaching to apple-health-grafana-ingester-1
apple-health-grafana-ingester-1  | unzipping the export file..
apple-health-grafana-ingester-1  | export file unzipped
apple-health-grafana-ingester-1  | influx is ready
apple-health-grafana-ingester-1  | loading workout routes
apple-health-grafana-ingester-1  | Traceback (most recent call last):
apple-health-grafana-ingester-1  |   File "//app.py", line 140, in <module>
apple-health-grafana-ingester-1  |     process_workout_routes()
apple-health-grafana-ingester-1  |   File "//app.py", line 92, in process_workout_routes
apple-health-grafana-ingester-1  |     for file in os.listdir(routes_path):
apple-health-grafana-ingester-1  |                 ^^^^^^^^^^^^^^^^^^^^^^^
apple-health-grafana-ingester-1  | FileNotFoundError: [Errno 2] No such file or directory: '/export/apple_health_export/workout-routes/'
apple-health-grafana-ingester-1 exited with code 1

~/src/apple-health-grafana main ⇡ 6s
❯ docker-compose ps
NAME                             IMAGE               COMMAND                  SERVICE             CREATED             STATUS              PORTS
apple-health-grafana-grafana-1   grafana/grafana     "/run.sh"                grafana             45 seconds ago      Up 43 seconds       0.0.0.0:3000->3000/tcp
apple-health-grafana-influx-1    influxdb:1.8        "/entrypoint.sh infl…"   influx              45 seconds ago      Up 43 seconds       8086/tcp
```

**Comment by k0rventen** [comment]
Hi, 

I just realised that if you do not have any workout routes (for example using an Apple Watch, or an outdoor running/walking app, this folder might not exist. I'll update the code accordingly to better handle this case (hopefully done by today).

Sorry for the inconvenience, hope you can try it once the issue is resolved :)

**Comment by k0rventen** [comment]
Hey, i just pushed fix 4ec26082f3e4b90a8f7996e9d54b20be330dfc02 which should resolve your issue !

If you can give it another go (remember to `docker-compose pull` beforehand to grab the latest image with the fix) and tell me how it went :smile:

**Comment by scarroll32** [comment]
That fixed it!
